openapi.json

```
@@ -22,7 +22,7 @@
         "backoff": {
             "initialInterval": 500,
             "maxInterval": 60000,
-            "maxElapsedTime": 3600000,
+            "maxElapsedTime": 300000,
```

```suggestion
            "maxElapsedTime": 900000
```
actually how about 15 mins? :)